Unit test suite throws an error attempting to redefine `EMPTY_TRASH_DAYS`

## Bug Description

<!-- Please describe clearly and concisely what the bug is. -->

If the constant `EMPTY_TRASH_DAYS` is not defined in the config or test config file, WordPress assigns a default value of `30` in the `default-contants.php` file.

https://github.com/WordPress/wordpress-develop/blob/95d26d682fb0fcb77fed308436e87914f7ce2e82/src/wp-includes/default-constants.php#L371-L373

The performance lab test suite attempts to define this value to zero in `WebP_Uploads_Load_Tests::it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero` causing an error to be thrown:



## Steps to reproduce

<!-- Please provide detailed steps on how to reproduce the bug. -->
1. Remove `EMPTY_TRASH_DAYS` from all test and general config files
2. Run `phpunit --filter force_deleted_and_empty_trash_days_is_set_to_zero` on the command line
3. phpunit should notify you of an error due when attempting to redefine the constant


## Textdumps

```
$ phpunit --filter force_deleted_and_empty_trash_days_is_set_to_zero
Installing...
Running as single site... To run multisite, use -c tests/phpunit/multisite.xml
Not running ajax tests. To execute these, use --group ajax.
Not running ms-files tests. To execute these, use --group ms-files.
Not running external-http tests. To execute these, use --group external-http.
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 7.8 seconds, Memory: 52.01 MB

There was 1 error:

1) WebP_Uploads_Load_Tests::it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero
Constant EMPTY_TRASH_DAYS already defined

/vagrant/content/plugins/performance-lab/tests/modules/images/webp-uploads/load-tests.php:279
```



## Additional Context

<!-- Please complete the following information. -->
 - PHP Version: 7.4
 - OS: Ubuntu
 - Browser: N/A
 - Plugin Version: `trunk` at 6c3e79573c39a3a87300c97405ac83f36d0aeee2
 - Device: N/A

<!-- Please add any additional information about the bug. -->

I'm running using a https://github.com/Chassis/Chassis box on VirtualBox with the following `config.local.yaml` file:

```yaml
# Domain names to use for Vagrant
hosts:
    - wp-perf.local

# Site Configuration
# (When overriding, include all values)
site:
    name: WP Perf Plugin

# Should we use multisite?
# Values: No, Yes, subdomains
multisite: No

# Plugins
plugins:
   - query-monitor
   - user-switching

# Extensions
extensions:
 - mailhog
 - imagick
 - phpunit
 - xdebug
 - memcache
 - tester

# PHP version
# Values: 5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0 (or 5.6.30)
php: 7.4

phpunit:
  version: 7.5
```

--- a/tests/modules/images/webp-uploads/load-tests.php
+++ b/tests/modules/images/webp-uploads/load-tests.php
@@ -229,11 +229,11 @@ class WebP_Uploads_Load_Tests extends ImagesTestCase {
 	}
 
 	/**
-	 * Remove the attached WebP version if the attachment is force deleted but empty trash day is not defined
+	 * Remove the attached WebP version if the attachment is force deleted
 	 *
 	 * @test
 	 */
-	public function it_should_remove_the_attached_webp_version_if_the_attachment_is_force_deleted_but_empty_trash_day_is_not_defined() {
+	public function it_should_remove_the_attached_webp_version_if_the_attachment_is_force_deleted() {
 		$attachment_id = $this->factory->attachment->create_upload_object(
 			TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/leafs.jpg'
 		);
@@ -248,35 +248,6 @@ class WebP_Uploads_Load_Tests extends ImagesTestCase {
 
 		$this->assertFileExists( path_join( $dirname, $metadata['sources']['image/webp']['file'] ) );
 		$this->assertFileExists( path_join( $dirname, $metadata['sizes']['thumbnail']['sources']['image/webp']['file'] ) );
-
-		wp_delete_attachment( $attachment_id, true );
-
-		$this->assertFileDoesNotExist( path_join( $dirname, $metadata['sizes']['thumbnail']['sources']['image/webp']['file'] ) );
-		$this->assertFileDoesNotExist( path_join( $dirname, $metadata['sources']['image/webp']['file'] ) );
-	}
-
-	/**
-	 * Remove the WebP version of the image if the image is force deleted and empty trash days is set to zero
-	 *
-	 * @test
-	 */
-	public function it_should_remove_the_webp_version_of_the_image_if_the_image_is_force_deleted_and_empty_trash_days_is_set_to_zero() {
-		$attachment_id = $this->factory->attachment->create_upload_object(
-			TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/leafs.jpg'
-		);
-
-		$file    = get_attached_file( $attachment_id, true );
-		$dirname = pathinfo( $file, PATHINFO_DIRNAME );
-
-		$this->assertIsString( $file );
-		$this->assertFileExists( $file );
-
-		$metadata = wp_get_attachment_metadata( $attachment_id );
-
-		$this->assertFileExists( path_join( $dirname, $metadata['sources']['image/webp']['file'] ) );
-		$this->assertFileExists( path_join( $dirname, $metadata['sizes']['thumbnail']['sources']['image/webp']['file'] ) );
-
-		define( 'EMPTY_TRASH_DAYS', 0 );
 
 		wp_delete_attachment( $attachment_id, true );
 

